imap_processing/cdf/config/imap_idex_l1a_variable_attrs.yaml

```
@@ -38,13 +38,13 @@ l1a_data_base: &l1a_data_base
 
 l1a_tof_base: &l1a_tof_base
   <<: *l1a_data_base
-  DEPEND_1: time_high_sr
-  LABL_PTR_1: time_high_sr_label
+  DEPEND_1: time_high_sample_res
```

should we call it this to match its description? or was that what IDEX wanted?
```suggestion
  DEPEND_1: time_high_sample_rate
```

---

OK. I named it _res short for resolution but I see what you mean now about how it mismatches the description.